docs/docs/references/embedding.mdx

```diff
@@ -69,32 +66,36 @@ Clicking `generate & copy url` will generate an embed URL based on the configura
 
 You can embed this one-off URL directly into your application, but you will manually need to update the URL each time the embed URL expires. Alternatively, you can add the [`code snippet`](#code-snippet) to your app to automatically generate embed URLs in your application.  
 
-### Code snippet
+## Code snippet
 
 Although you can generate URLs directly from Lightdash with a long expiration, it is recommended to generate your own JWT embed tokens in your backend with a short expiration using your `secret` to make sure people can't be using embed URLs outside your app. 
 
 To make this easier to integrate, we included some code snippets you can copy and use in your app to generate a valid embed URL.
 
-### Passing external variables to user attributes
+<img src={EmbedDeveloper} width="668" height="466" style={{display: "block", margin: "0 auto 20px auto"}}/>
 
-For embedded content, you can either set user attributes to constant values, or you can adjust the value of the user attribute based on a variable coming from your app. 
+### User metadata
 
-For example, if I want users in my app to only see data filtered to their company, I can use a variable user attribute. 
+You can pass user metadata (specifically, an external user ID) from your application so that anytime someone views your embedded dashboard and runs queries in Lightdash, these query logs are enriched with this user metadata. Specifcally, this user metadata gets added to the [query tags](/references/usage-analytics.mdx#query-tags) for queries run in Lightdash.
```

Comment:
just so you know, on bigquery, only lowercase alphanumeric number (and _) are allowed on externalIds... not sure if this is something we should add to the docs, or just keep in mind. https://lightdash.slack.com/archives/C069QF4UTL5/p1704816492099229

Reply:
Is this just in BigQuery, or in all data warehouses?